web/experimental/embed-new.html

```diff
@@ -35,29 +35,23 @@
 </head>
 
 <body class="d-flex flex-column">
-
-<nav id="navbar" class="UnderlineNav p-2">
-    <div class="UnderlineNav-body">
-        <div class="BtnGroup">
-            <button id="editor-tab" class="btn BtnGroup-item selected" selected type="button">Your code</button>
-            <button id="test-tab" class="btn BtnGroup-item" type="button">Test code</button>
-            <button id="console-tab" class="btn BtnGroup-item" type="button">
-                Output
-                <span id="unread-console-counter" class="Counter" hidden></span>
-            </button>
-        </div>
-    </div>
-    <div class="UnderlineNav-actions">
-        <button id="show-hint" class="btn disabled">Hint</button>
-        <button id="format-code" class="btn">Format</button>
-        <button id="reload-gist" class="btn mr-2">Reset</button>
-        <button id="execute" class="btn btn-primary">
-            <div class="octicon medium-octicon octicon-triangle-right"></div>
-            Run
+<div id="navbar" class="tabnav dp-tabs">
```

Comment:
Why is the definition order reversed? Are we using some left to right layout option?

Reply:
Gotcha. If doing this myself I would probably have used flexbox, but the change works either way.

Reply:
I'm also a +1 for using flexbox here, since it's how everything else in this UI is positioned, just about.